summerofcode/2023.md

```diff
@@ -141,3 +141,34 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Difficulty: Medium
 - Upstream Issue (URL): https://github.com/strimzi/strimzi-kafka-operator/issues/8030
 
+### Curve
+
+#### Support metadata of Curve Block Storage stored in the database
+
+- Description: Currently the [metadata](https://github.com/opencurve/curve/blob/master/src/mds/nameserver2/namespace_storage.cpp) of current CurveBS is stored in [Etcd](https://github.com/opencurve/curve/tree/master/src/kvstorageclient). In order to apply to more scenarios, it is hoped that the metadata can add a sql database like mysql, MariaDB, PostgreSQL, etc. as one of the storage engines. Add a new sqlstorageclient for [metadata](https://github.com/opencurve/curve/blob/master/src/mds/nameserver2/namespace_storage.cpp).
+- Expected outcome: It is hoped that the user can choose whether the metadata is stored in the kv engine or the sql engine through the configuration file.
+- Recommended Skills: C++, SQL
+- Mentor(s): Xiaocui Li @ilixiaocui
+- Expected project size: 175h
+- Difficulty: Medium
+- Upstream Issue (URL): https://github.com/opencurve/curve/issues/2224
+
+#### Add compression feature for Curve snapshot
+
+- Description: Curve snapshots are currently uploaded to s3, and no compression is used. Using compression can greatly reduce the space occupied by snapshots, thereby saving storage costs.
+- Expected outcome: 1） Detailed Design Documentation 2） Realize curve snapshot compression upload and download decompression 3） Add unit tests and integration test cases 4） Merge your PR into the opencurve repo
+- Recommended Skills: C++
+- Mentor(s): Chaojie Xu @xu-chaojie
```

Comment:
I haven't seen mentor mentioned here in the https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L1093 file but I see the PR author's name, so, we should trusting the judgement of a project maintainer to list a community member as a mentor. I've also seen contributions from the mentor to the project.

So, 👍

Reply:
Oh, but, I just noticed the email addresses for the mentors are missing. Can you add those @ilixiaocui ?

Reply:
+1, since a maintainer is making the proposal we can accept it even though they aren't participating directly as a mentor.